packages/ppx_monadic/ppx_monadic.1.0.4/opam

```diff
@@ -0,0 +1,27 @@
+opam-version: "1.2"
+version: "1.0.4"
+authors: "Jun Furuse"
+maintainer: "jun.furuse@gmail.com"
+homepage: "https://bitbucket.org/camlspotter/ppx_monadic"
+bug-reports: "https://bitbucket.org/camlspotter/ppx_monadic/issues?status=new&status=open"
+dev-repo: "hg://https://bitbucket.org/camlspotter/ppx_monadic"
+build: [
+  [ "ocaml" "setup.ml" "-configure" "--prefix" prefix ]
+  [ "ocaml" "setup.ml" "-build" ]
+]
+install: [
+  [ "ocaml" "setup.ml" "-install" ]
+]
+remove: [
+  [ "ocaml" "setup.ml" "-configure" "--prefix" prefix ]
+  [ "ocaml" "setup.ml" "-uninstall" ]
+]
+depends: [
+  "ocamlfind" { build }
+  "omake" { build }
+  "ppx_tools"
+  "ppxx" { = "1.3.0" }
+]
+available: [
+  ocaml-version = "4.03.0"
```

Comment:
Will this definitely not work with 4.03.1dev for example? We prefer lower bounds where possible or else it becomes hard to test with newer versions of the compiler (which may not change the AST and so should work). ppx_tools also provides a more compatible interface across compiler revisions.